### PR TITLE
Computing for Engineers with Julia [CEJ]

### PR DESCRIPTION
**Pull request by clstaudt**
New course: Computing for Engineers with Julia

### CODE REVIEW

File: notebooks/.assets/images/art-computing-for-engineers-with-julia.png
Comment (flokru) [inline notebooks/.assets/images/art-computing-for-engineers-with-julia.png:1]: 😃👍

Comment (clstaudt) [reply notebooks/.assets/images/art-computing-for-engineers-with-julia.png:1]: Got a few more... should we add them?

data-analysis-with-python-2
data-analysis-with-python
deep-learning-with-tensorflow
machine-learning-with-python

Comment (flokru) [reply notebooks/.assets/images/art-computing-for-engineers-with-julia.png:1]: I think they're all great, but others should decide that.

Comment (clstaudt) [reply notebooks/.assets/images/art-computing-for-engineers-with-julia.png:1]: #6

**Comment by clstaudt** [comment]
@flokru   This is ready to merge in my opinion.

**Comment by flokru** [comment]
I am honored to be added as a reviewer, but I fear I am too far out of topic to add anything meaningful here. 😅

**Review by ccauet — APPROVED** [review]
Looks good to me

**Pull request merged**